worldedit-core/src/main/java/com/sk89q/worldedit/regions/CuboidRegion.java

```diff
@@ -34,6 +34,29 @@
 
 /**
  * An axis-aligned cuboid. It can be defined using two corners of the cuboid.
+ *
+ * Example usage:
+ * <pre>
+ * <code>
+ * Bukkit.getScheduler().runTaskTimerAsynchronously(this, () -> {
+ *    for (final Pair<Cuboid, List<WantedMaterial>> mine : mines.values()) {
+ *        final World world = BukkitAdapter.adapt(mine.getKey().getWorld());
+ *
+ *        final CuboidRegion selection = new CuboidRegion(
+ *            world,
+ *            toBlockVector(mine.getKey().getLowerNE()),
+ *            toBlockVector(mine.getKey().getUpperSW())
+ *        );
+ *
+ *        try (final EditSession session = WorldEdit.getInstance().newEditSessionBuilder().world(world).build()) {
```

Comment:
This is more showing how to perform a random pattern edit, which is much better covered by the docs than this